.github/e2e-tests.yml

```diff
@@ -1019,6 +1019,34 @@ runner-test-matrix:
 
   # START: CCIPv1.6 tests
 
+  - id: smoke/ccip/ccip_reorg_test.go:LessThanFinalityTests
+    path: integration-tests/smoke/ccip/ccip_reorg_test.go
+    test_env_type: docker
+    runs_on: ubuntu-latest
+    triggers:
+      - PR E2E Core Tests
```

Comment:
would you like to keep it running in nightly for few days to check for  flakeyness before enabling in PR?

Reply:
Good note, gonna keep it in nightly for now.